Formula/marcli.rb

```diff
@@ -0,0 +1,28 @@
+class Marcli < Formula
+  desc "Parse MARC (ISO 2709) files"
+  homepage "https://github.com/hectorcorrea/marcli"
+  url "https://github.com/hectorcorrea/marcli/archive/refs/tags/1.0.1.tar.gz"
+  sha256 "abea9ae6a7cd2d6874ac71d0aa429b9eab367daa4f301161a0671f85608216cf"
+  license "MIT"
+  head "https://github.com/hectorcorrea/marcli"
+
+  depends_on "go" => :build
+
+  resource("testdata") do
```

Comment:
```suggestion
  resource "testdata" do
```
Let's follow the format used by (most) other formulae here.

Reply:
Just FYI, I wrote it this way because the Formula Cookbook uses parentheses in its example: https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula